device/isis/routerange.yaml

```diff
@@ -0,0 +1,41 @@
+components:
+  schemas:
+    Isis.RouteRange:
```

Comment:
Since we support multi-topology capability at the router, we should also have ability to enable route advertisement per MT. These two should be supported together.

Reply:
We can add this feature latter on.